content/docs/user-guide/how-to/add-output-to-stage.md

```diff
@@ -0,0 +1,47 @@
+# Add Output to Stage
+
+There are situations where we execute a stage without specifying all the
+<abbr>outputs</abbr> in `dvc.yaml` (either via `dvc run` or manually), and then
+later want to add more outputs to that stage.
+
+Re-executing a stage using `dvc run -f` to add an output is an expensive step,
+as it requires time and resources. Follow the steps below to add an output to
+the stage without executing it again.
+
+We start with an example `prepare` stage in `dvc.yaml`, which has a single
+output. To add a missing output to this stage, we can edit `dvc.yaml` like this:
+
+```git
+ stages:
+   prepare:
+     cmd: python src/prepare.py
+     deps:
+     - src/prepare.py
+     outs:
+     - data/train
++    - data/validate
+```
+
+We have added `data/validate` as an output to the `prepare` stage.
```

Comment:
Not sure about the usefulness of this sentence, maybe just mention it in the previous paragraph before the code block.